### DISCUSSION TITLE
State of NuGet Blog

### DISCUSSION THREAD

**Discussion by jcjiang** [NuGet/Home#10600] [General]
Have thoughts after reading the State of NuGet blogpost? Share here.

**Comment by Rand-Random** [comment]
Just in case you missed the blog, like I did, here is the link
https://devblogs.microsoft.com/nuget/state-of-the-nuget-ecosystem/

**Comment by brettrowberry** [comment]
One thing I struggle with is that the .NET SDK isn’t a direct replacement for nuget.exe. I think it should be.

**Reply by JonDouglas** [reply]
@brettrowberry That is definitely one of our goals. What are the biggest areas that are missing parity today?

**Reply by brettrowberry** [reply]
Don’t remember. I think it was creating packages from multiple projects.

**Reply by Rand-Random** [reply]
> @brettrowberry That is definitely one of our goals. What are the biggest areas that are missing parity today?

For me it is the inability to use commands without project/solution eg. add/init/install commands.

That‘s the main reason why I used nuget.exe for my reported issues here
https://github.com/NuGet/Home/discussions/10579
https://github.com/NuGet/Home/issues/10593
https://github.com/NuGet/Home/issues/10618

I want to work directly with nuget packages and not with projects.

For one job neither dotnet nor nuget was enough and I had to use https://github.com/dotnet-outdated/dotnet-outdated

I needed a „dotnet restore“ for all packages with the PackageId „Foo.*“

**Reply by JonDouglas** [reply]
@brettrowberry This one? https://github.com/NuGet/Home/issues/3891

**Reply by JonDouglas** [reply]
@Rand-Random Although we've responded to a number of those issues already, you're basically asking to separate packages from projects/solution? Maybe a new discussion would help us discuss that further.

As for the dotnet-outdated project, what aspects of that tool would be beneficial if NuGet supported them in dotnet CLI or Visual Studio? Would love another issue or discussion talking about your ideas here. For example there is https://github.com/NuGet/Home/issues/4103 & various list commands to check deprecation/vulnerabilities.

**Reply by brettrowberry** [reply]
Here’s another https://github.com/NuGet/Home/issues/10398
